charts/stable/unifi-poller/values.yaml

```diff
@@ -6,15 +6,18 @@
 #
 
 image:
+  # -- Image to deploy.
   repository: golift/unifi-poller
+  # -- Image [k8s pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images).
   pullPolicy: IfNotPresent
-  tag: 2.0.1
+  # -- Image tag to deploy.
+  tag: 2.1.3
 
-strategy:
-  type: Recreate
+# -- [Deployment strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) used to replace old Pods by new ones.
+strategy: Recreate
```

Comment:
```suggestion
```

This option was changed in the common v3 library. Sane defaults are now properly determined in there, so we can safely remove it from the subchart values.